isis/src/rosetta/apps/rososiris2isis/osirisFilters.trn

```diff
@@ -0,0 +1,148 @@
+#
```

Comment:
Does this translation file belong at this location, or in `$ISIS3DATA/rosetta/translations` ? 

The actual file looks good! :+1: